docs/reference/commandline/cli.md

```diff
@@ -46,6 +46,8 @@ by the `docker` command line:
   unsuitable for Docker.
 * `DOCKER_RAMDISK` If set this will disable 'pivot_root'.
 * `DOCKER_TLS_VERIFY` When set Docker uses TLS and verifies the remote.
+* `DOCKER_TRUST` When set Docker uses notary to sign and verify images.
+  Equates to `--untrusted=false` for build, create, pull, push, run.
```

Comment:
I agree that we shouldn't publicize that variable for now. Let's wait until trusted is the default behavior.